app/views/layouts/_navigation.html.erb

```diff
@@ -1,5 +1,10 @@
-<nav class="navbar navbar-default">
+<% if user_signed_in? %>
+  <nav class="navbar navbar-default <%= current_user.theme %>">
+<% else %>
+    <nav class="navbar navbar-default">
+<% end %>   
```

Comment:
These 5 lines can be condenced into the following single line.

```
<nav class="navbar navbar-default <%= current_user.try(:theme) %>">
```

That will put the theme if possible, if not it will be blank.